modules/getting-started-openshift-gitops.adoc

```diff
@@ -0,0 +1,128 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/working-with-openshift-gitops.adoc
+
+[id="getting-started-with-red-hat-openshift-gitops_{context}"]
+= Getting Started with Red Hat OpenShift GitOps
```

Comment:
This is a collection of procedure modules, we will need to create an assembly for this and add the different modules to it. We will also need to add an entry to the Topic map file.